android/src/main/java/com/swmansion/reanimated/nodes/PropsNode.java

```diff
@@ -73,28 +93,17 @@ protected Double evaluate() {
             dest = jsProps;
           }
           ReadableType type = style.getType(key);
-          switch (type) {
-            case Number:
-              dest.putDouble(key, style.getDouble(key));
-              break;
-            case String:
-              dest.putString(key, style.getString(key));
-              break;
-            case Array:
-              dest.putArray(key, (WritableArray) style.getArray(key));
-              break;
-            default:
-              throw new IllegalArgumentException("Unexpected type " + type);
-          }
+          addProp(dest, key, node.value());
```

Comment:
I just tested this branch and it works great for my use case but how can I test that this particular refactoring doesn't cause any regression?

Reply:
I don't think this is going to work. Doesn't seem that the two implementations are compatible here. node.value() in this case won't get the individual style prop values

Reply:
yes that makes sense 👍🏻